### ISSUE TITLE
Bracket Highlight

### ISSUE THREAD

**Issue by Nabawe** [adamjsx/arctis#5]
Hi again, since I was here I might as well comment on a small detail, hope I am not misusing the issues tabs for this one.

"editor.matchBrackets": "near"

When I am with rested eyes this looks perfect but if I am a tad tired and there are many braces one next to each other the extra lines drawn from the "highlight box" make things harder to distinguish.

I apologice that I can't make a suggestion on how to solve this, but it should be something a bit more soft ( Alpha ? ) but keep just enough sharpness that it is still helpful, I don't know if using the old highlight method could help ( typical background change, or filled box, again may be more transparent ) or to get an option to choose from.

To sum up the empty box you are using feels better but it needs to be a bit softer and may be use another color or line style.

Again thanks for your time.

![bracket highlight](https://user-images.githubusercontent.com/11269209/167303461-5e138b80-f12e-41d9-987e-080c38440f7e.png)
![old example from Andromeda theme](https://user-images.githubusercontent.com/11269209/167303462-44fb3ae3-fcd9-4882-a4f6-ff92ef0a238c.png)

**Comment by adamdotjs** [comment]
Great suggestion. That's how it was with Nord and I never bothered to change it. I'll take a look.
